forge/ee/routes/flowBlueprints/index.js

```diff
@@ -45,14 +49,19 @@ module.exports = async function (app) {
             reply.send(flowTemplates)
         } else {
             // get all flow templates - typically for administration purposes
-            const flowTemplates = await app.db.models.FlowTemplate.getAll(paginationOptions, filter)
+            const flowTemplates = await app.db.models.FlowTemplate.getAll(paginationOptions, filter, { includeFlows: false })
             flowTemplates.blueprints = flowTemplates.templates.map(ft => app.db.views.FlowTemplate.flowBlueprintSummary(ft))
             reply.send(flowTemplates)
         }
     })
 
     app.get('/:flowBlueprintId', {
-        preHandler: app.needsPermission('flow-blueprint:read'),
+        preHandler: async (request, reply) => {
```

Comment:
same here? Not that there's anything wrong with checking it's just that I don't see the point.

Reply:
As above, this is needed for the Add BluePrint to existing instance work